.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push]
+
+env:
+  DEVELOPER_DIR: /Applications/Xcode_11.2.app/Contents/Developer
+  WORKSPACE: ReactiveLists.xcworkspace
+  LIBRARY_SCHEME: ReactiveLists
+  EXAMPLE_APP_SCHEME: Example
+  IOS_SDK: iphonesimulator13.2
+
+jobs:
+  iOS:
+    name: iOS Test
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        destination: ["OS=13.2,name=iPhone 11"]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Bundle Install
+        run: bundle install
+      - uses: actions/cache@v1
```

Comment:
This is the relevant change, right? Any docs you can point me to?

Reply:
Yup.  You can see more info here: https://github.com/actions/cache

Its a marketplace action so I was able to just `uses` it.  There are examples in their docs for how to use it for CocoaPods projects so I lifted the configuration from [there](https://github.com/actions/cache/blob/master/examples.md#swift-objective-c---cocoapods).